katas/langtonant/williams/langtonant.test.js

```diff
@@ -0,0 +1,5 @@
+import { ant } from './langtonant';
+
+test('Step antX', () => {
```

Comment:
The name of the test can be improved adding context and what is the expected that it is going to validate